src/lib/features/project/project-controller.ts

```diff
@@ -77,6 +77,16 @@ export default class ProjectController extends Controller {
                     summary: 'Get a list of all projects.',
                     description:
                         'This endpoint returns an list of all the projects in the Unleash instance.',
+                    parameters: [
```

Comment:
I guess we'll need to repeat it in the enterprise since this method is duplicated

Reply:
in Enterprise there is this comment:
https://github.com/ivarconr/unleash-enterprise/blob/fa284169f6f4a282d9a26bfed18aa2a5ba8346c3/src/project/project-controller.ts#L123
```
// this operation is also described in OSS, so that operation will take precedence in the output. In other words, anything you put here is ignored.
```